src/components/search/SearchBar.tsx

```diff
@@ -48,31 +43,54 @@ const SearchBar = ({ onCardClick }: { onCardClick?: () => void }) => {
 
       return () => clearTimeout(timeoutId);
     }
-    setSearchedVideos(null);
+    setSearchedVideos([]);
   }, [searchTerm, fetchData]);
 
-  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    setSearchTerm(event.target.value);
-  };
-
-  const clearSearchTerm = () => {
-    setSearchTerm('');
-  };
-
-  const handleCardClick = (videoUrl: string) => {
-    if (onCardClick !== undefined) {
-      onCardClick();
-    }
-    clearSearchTerm();
-    router.push(videoUrl);
-  };
+  useEffect(() => {
+    const handleKeyPress = (event: KeyboardEvent) => {
+      switch (event.code) {
+        case 'KeyK':
+          if (event.ctrlKey) {
```

Comment:
You are just checking with ctrl key but as the name suggests it should also support `⌘K` on mac